src/elements/pearl-chain/pearl-chain-leg/pearl-chain-leg.visual.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../../core/testing/private.js';
+
+import './pearl-chain-leg.js';
+
+describe('sbb-pearl-chain-leg', () => {
```

Comment:
I recommend to share the template either with cases and describeEach, or just a beforEach(). now e.g. every departure and arrival time is the same (times don't affect rendering right now)